openedx/core/djangoapps/content_libraries/tests/test_libraries_index.py

```diff
@@ -6,6 +6,7 @@
 from django.core.management import call_command
 from django.test.utils import override_settings
 from mock import patch
+from search.search_engine_base import SearchEngine
```

Comment:
Looks like a duplicate import (imported two lines later too)

Reply:
I can't wait until we get `black` and `isort` going on platform :)